.github/workflows/python.yaml

```diff
@@ -18,7 +18,7 @@ on:
 jobs:
   coverage:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Updated to ubuntu-22.04 because the python dependencies build procedures are not portable and no wheels are available for newer OS versions (pip install fails on ubuntu-24.04 with default python version).

Wheels are available for ubuntu-22.04.